lib/client.js

```diff
@@ -212,6 +212,7 @@ Client.prototype._setSequenceArgs = function(argsScheme, args) {
     return args;
   }
   for (var partIndex in argsScheme) {
+    partIndex = partIndex.replace('[]','')
```

Comment:
Well, I agree it's a very specific issue, then i forked and create my own npm package to solve urgent my issue, but I decided to pull request it anyway.
I think it's a error with the SOAP server, but I have an argScheme with brackets while it should be just a string. If I try to pass the arg with the bracket the application crashes